internal/sdks/sdk_instructions/android.md

```diff
@@ -0,0 +1,124 @@
+# Installation steps
+If you want to skip ahead, the final code is available in our [GitHub repository](https://github.com/launchdarkly/hello-android).
+
+1. Open Android Studio and create a new project named `hello-android` with an empty activity.
+
+2. Next, add the LaunchDarkly SDK as a dependency in the `app/build.gradle` file:
+```java
+dependencies {
+  ...
+  implementation("com.launchdarkly:launchdarkly-android-client-sdk:5.1.1")
+}
+```
+
+3. Open the file `MainActivity.kt` and add the following code:
+```java
+import com.launchdarkly.sdk.android.LDClient
+
+class MainActivity : AppCompatActivity() {
+
+  // Set BOOLEAN_FLAG_KEY to the boolean feature flag you want to evaluate.
+  val BOOLEAN_FLAG_KEY = "my-flag-key"
```

Comment:
Looks like we still have [this bug](https://app.shortcut.com/launchdarkly/story/239265/should-not-find-replace-launchdarkly-flag-key-in-python-sdk-instructions) in a couple places. We don't need to fix that in this PR, but could you fix it in a separate one?